packages/map-template/src/components/MapTemplate/MapTemplate.jsx

```diff
@@ -230,7 +227,7 @@ function MapTemplate({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId, p
         }
 
         setHasDirectionsOpen(currentAppView === appStates.DIRECTIONS);
-        _locationsDisabled = currentAppView === appStates.DIRECTIONS;
+        locationsDisabledRef.current = currentAppView === appStates.DIRECTIONS;
```

Comment:
Do we need both the `locationsDisabled` useRef and the `hasDirectionsOpen` useState? They contain the same information.

Reply:
Good point, we might be able to get rid of one of them. In this case, should we keep the useRef instead of the useState?

Reply:
I think the useRef works better.

Reply:
Refactored the code and removed the `hasDirectionsOpen` state.